api-service/src/controllers/ConnectorInstanceCreate/connectorInstanceCreate.ts

```diff
@@ -0,0 +1,38 @@
+import _ from "lodash";
```

Comment:
File name should start with capital C only

Reply:
change done